recipe/meta.yaml

```diff
@@ -81,13 +96,14 @@ outputs:
         # expat here is _only_ required for mesa-dri-drivers
         - {{ cdt('expat') }}                 # [linux]
         - {{ cdt('libselinux-devel') }}      # [linux]
-        - pkg-config
-        - m4
-        - autoconf
-        - automake
+        - {{native}}pkg-config
```

Comment:
m2w64-pkg-config is for finding mingw-w64 libraries, I am not sure that's what you want here, probably not.

Reply:
I'll change to m2-pkg-config. Thanks.